### ISSUE TITLE
Service Layer usage/role

### ISSUE THREAD

**Issue by carloperocchio** [rvilalta/ietf-te-path-computation#18]
Questions and answers discussion: Italo Busi, Sergio Belotti, Daniele Ceccarelli, Francesco Lazzeri, Gianmarco Bruno, Carlo Perocchio
Monday, December 05, 2016

Q: Which is the usage/role of service layer within the compute request?

Context: PC-API

R: optional parameter. It is the possibility to identify the layer (technology) at which the tunnel is requested.

**Comment by italobusi** [comment]
Italo/Sergio/Ricard:

serviceLayer is remained as inheritance from ONF

The TE Tunnel model allows to specify the topology-id, which can be used to infer the layer for the Tunnel if and only if the topology is at a single layer
Need to further discuss with the TE Tunnel Design Team. PC-API will align with any change in the TE Tunnel

@rvilalta : Please comment serviceLayer

**Comment by italobusi** [comment]
The serviceLayer has been removed.

In TE Tunnel model I have found the following attributes:

```
       |  |  |  +--rw encoding?             identityref
       |  |  |  +--rw switching-type?       identityref

```
They are defined within the tunnel-p2p-params_config grouping which is not used by Path Computation RPC and it seems providing more attributes than needed by Path Computation RPC

To be further discussed with the TE Tunnel Design Team

**Comment by sergiobelotti** [comment]
The issue is closed sicne the issue is covered in #31
